### PR TITLE
mention MariaDB

### PR DESCRIPTION
**Pull request by robertsilen**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/mysql_replication.py [hunk 1]
```diff
@@ -191,7 +191,8 @@
     version_added: '0.1.0'
 
 notes:
-- If an empty value for the parameter of string type is needed, use an empty string.
+   - Compatible with MariaDB or MySQL
```
Comment (Andersson007) [inline plugins/modules/mysql_replication.py]:
```suggestion
   - Compatible with MariaDB or MySQL.
```

File: plugins/modules/mysql_user.py [hunk 2]
```diff
@@ -188,6 +188,7 @@
     version_added: '3.9.0'
 
 notes:
+   - Compatible with MySQL or MariaDB
```
Comment (Andersson007) [inline plugins/modules/mysql_user.py]:
```suggestion
   - Compatible with MySQL or MariaDB.
```

File: plugins/modules/mysql_variables.py [hunk 3]
```diff
@@ -54,6 +54,9 @@
   description: Complete reference of the MySQL SET command documentation.
   link: https://dev.mysql.com/doc/refman/8.0/en/set-statement.html
 
+notes:
+   - Compatible with MariaDB or MySQL
```
Comment (Andersson007) [inline plugins/modules/mysql_variables.py]:
```suggestion
   - Compatible with MariaDB or MySQL.
```

**Review by Andersson007 — COMMENTED** [review]
@robertsilen LGTM, just a could of nits

**Comment by Andersson007** [comment]
@robertsilen also please take a look at the sanity failures

**Comment by robertsilen** [comment]
> @robertsilen also please take a look at the sanity failures

@Andersson007 fine now

**Review by Andersson007 — APPROVED** [review]

**Comment by Andersson007** [comment]
@robertsilen thanks for the contribution!

**Comment by robertsilen** [comment]
> @robertsilen thanks for the contribution!

Thanks! When will the update appear at https://docs.ansible.com/ansible/latest/collections/community/mysql/index.html ?

**Comment by Andersson007** [comment]
> > @robertsilen thanks for the contribution!
> 
> Thanks! When will the update appear at https://docs.ansible.com/ansible/latest/collections/community/mysql/index.html ?

Some time after we release the collection. Not sure when exactly as there are a couple of pending PRs we plan to merge before the release.

**Pull request merged**